properties.schema

```diff
@@ -419,84 +429,84 @@
             "default": "#36CDE8",
-            "help": "Notify background colour"
+            "help": "Sets the Notify (pop up) colour."
           },
           "notify-inverted": {
             "title": "Notify colour - inverted",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#FFFFFF",
-            "help": "Notify foreground colour"
+            "help": "Defines the Notify text colour. Should be a colour that provides good contrast against the Notify colour."
           },
           "notify-link": {
             "title": "Notify link colour",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify link font colour"
+            "help": "Defines the Notify link font colour."
           },
           "notify-link-hover": {
             "title": "Notify link colour - hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify link font colour when hovered over"
+            "help": "Defines the Notify link font colour when hovered over."
           },
           "notify-btn": {
             "title": "Notify button colour",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify button background colour"
+            "help": "Defines the Notify Btn colour."
           },
           "notify-btn-inverted": {
             "title": "Notify button colour - inverted",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify button foreground colour"
+            "help": "Defines the Notify Btn text / icon colour. Should be a colour that provides good contrast against the Notify Btn colour."
           },
           "notify-btn-hover": {
             "title": "Notify button colour - hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify button backgroound colour when hovered over"
+            "help": "Defines the Notify Btn colour when hovered over."
           },
           "notify-btn-inverted-hover": {
             "title": "Notify button colour - inverted hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify button foreground colour when hovered over"
+            "help": "Defines the Notify Btn text / icon colour when hovered over. Should be a colour that provides good contrast against the Notify Btn hover colour."
           },
           "notify-icon": {
             "title": "Notify icon colour",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify icon background colour"
+            "help": "Defines the Notify Icon colour."
```

Comment:
I'm not sure this help text really adds anything as the title is sufficiently clear by itself.